docs/overview/office-add-in-availability.md

```diff
@@ -37,6 +38,7 @@ To work as expected, your Office Add-in might depend on a specific Office host,
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.7</a><br>
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.8</a><br>
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.9</a><br>
+        - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">CustomFunctionsRuntime 1.1</a><br>
```

Comment:
Would be nice to deep link this. When I clicked and saw top of the page I was like "where's custom functions"? Is there a way to give it an H2 so you can deep link to the custom functions section?